{"priority": "minor", "kind": "bug", "repository": {"links": {"self": {"href": "data/repositories/osrf/gazebo.json"}, "html": {"href": "#!/osrf/gazebo"}, "avatar": {"href": "data/bytebucket.org/ravatar/{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}ts=1694483"}}, "type": "repository", "name": "gazebo", "full_name": "osrf/gazebo", "uuid": "{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}"}, "links": {"attachments": {"href": "data/repositories/osrf/gazebo/issues/2038/attachments_page=1.json"}, "self": {"href": "data/repositories/osrf/gazebo/issues/2038.json"}, "watch": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/gazebo/issues/2038/watch"}, "comments": {"href": "data/repositories/osrf/gazebo/issues/2038/comments_page=1.json"}, "html": {"href": "#!/osrf/gazebo/issues/2038/flaky-seg-fault-in-simbody-portion-of"}, "vote": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/gazebo/issues/2038/vote"}}, "reporter": {"display_name": "Steve Peters", "uuid": "{2ccfed09-18b8-4921-8d58-15ef01092802}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B2ccfed09-18b8-4921-8d58-15ef01092802%7D"}, "html": {"href": "https://bitbucket.org/%7B2ccfed09-18b8-4921-8d58-15ef01092802%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/1fb4816dad9e68337d3096f750951f6cd=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsSP-1.png"}}, "nickname": "Steven Peters", "type": "user", "account_id": "557058:5de38267-b118-494c-aa76-4fab35448816"}, "title": "Flaky seg-fault in simbody portion of REGRESSION_1702_remove_model_scene_nodes", "component": {"name": "rendering", "links": {"self": {"href": "data/repositories/osrf/gazebo/components/122881.json"}}}, "votes": 0, "watches": 1, "content": {"raw": "I've been seeing repeatable intermittent seg-faults in the simbody instantiation of the `REGRESSION_1702_remove_model_scene_nodes` test (ie. `PhysicsEngines/Issue1702Test.SpawnDeleteSpawnAgain/2`) on the default branch. It consistently seg-faults in [Visual::GetBoundsHelper](#!/osrf/gazebo/src/bb6cc0893c9b6b160d46734c4fe9dbcbed1a1656/gazebo/rendering/Visual.cc?fileviewer=file-view-default#Visual.cc-1946) when trying to call `node->_updateBounds()` on the `OgreSceneNode`.\r\n\r\n[A full backtrace is here](https://gist.github.com/scpeters/b651d1e67c6d2b8676359290ab16c766).\r\n\r\nA quick check of recent jenkins results shows the test also failing on the simbody test on gazebo7:\r\n\r\n* [console log](http://build.osrfoundation.org/job/gazebo-ci-gazebo7-trusty-amd64-gpu-nvidia/70/consoleText)\r\n\r\nAny ideas @iche033 ?", "markup": "markdown", "html": "<p>I've been seeing repeatable intermittent seg-faults in the simbody instantiation of the <code>REGRESSION_1702_remove_model_scene_nodes</code> test (ie. <code>PhysicsEngines/Issue1702Test.SpawnDeleteSpawnAgain/2</code>) on the default branch. It consistently seg-faults in <a data-is-external-link=\"true\" href=\"#!/osrf/gazebo/src/bb6cc0893c9b6b160d46734c4fe9dbcbed1a1656/gazebo/rendering/Visual.cc?fileviewer=file-view-default#Visual.cc-1946\" rel=\"nofollow\">Visual::GetBoundsHelper</a> when trying to call <code>node-&gt;_updateBounds()</code> on the <code>OgreSceneNode</code>.</p>\n<p><a data-is-external-link=\"true\" href=\"https://gist.github.com/scpeters/b651d1e67c6d2b8676359290ab16c766\" rel=\"nofollow\">A full backtrace is here</a>.</p>\n<p>A quick check of recent jenkins results shows the test also failing on the simbody test on gazebo7:</p>\n<ul>\n<li><a data-is-external-link=\"true\" href=\"http://build.osrfoundation.org/job/gazebo-ci-gazebo7-trusty-amd64-gpu-nvidia/70/consoleText\" rel=\"nofollow\">console log</a></li>\n</ul>\n<p>Any ideas @iche033 ?</p>", "type": "rendered"}, "assignee": null, "state": "new", "version": {"name": "all", "links": {"self": {"href": "data/repositories/osrf/gazebo/versions/225575.json"}}}, "edited_on": null, "created_on": "2016-08-26T18:18:11.764506+00:00", "milestone": null, "updated_on": "2016-09-14T19:19:59.822191+00:00", "type": "issue", "id": 2038}